oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -501,16 +501,21 @@ impl Runtime {
     }
 
     /// Returns the least restrictive (i.e. least confidential, most trusted) label that this Node
-    /// may downgrade to. This takes into account all the [downgrade privilege](NodeInfo::privilege)
-    /// that the node possesses.
-    fn get_node_downgraded_label(&self, node_id: NodeId) -> Label {
-        // Original (static) Node label.
-        let node_label = self.get_node_label(node_id);
+    /// may downgrade `initial_label` to. This takes into account all the [downgrade
+    /// privilege](NodeInfo::privilege) that the node possesses.
+    ///
+    /// When reading from a Channel, `initial_label` is the Label of the Channel from which the Node
+    /// is reading: the thing being downgraded is the data being read into the Node (protected by
+    /// the Channel Label).
+    ///
+    /// When writing to a Channel, `initial_label` is the Label of the Node itself: the thing being
+    /// downgraded is the data inside the Node (protected by the Node Label).
```

Comment:
nit: the latter two paragraphs describe how some other code might use this function, not what the function does, so would probably be better moved to the relevant invocation points.

Reply:
Done.